### DISCUSSION TITLE
KERI Implementor's Call 08-13-2024

### DISCUSSION THREAD

**Discussion by pfeairheller** [WebOfTrust/keri#94] [Meetings]
### Zoom
https://us06web.zoom.us/j/81679782107?pwd=cTFxbEtKQVVXSzNGTjNiUG9xVWdSdz09

### Reports
KERIpy - @m00sey (main), @pfeairheller (back up)
SignifyPy - @pfeairheller , @m00sey (back up)
KERIA - @rodolfomiranda , @2byrds (backup)
Signify-TS - @kentbull , @daniellenskjö (backup)
Cesride - @kentbull , @daidoji (backup)
Kerific - @kordwarshuis
kerisse - @henkvancann
keridoc - @henkvancann
signify-browser-extension - @2byrds , @rodolfomiranda (backup)
polaris-web - @2byrds , @rodolfomiranda (backup)

### Discussion
**Delegator approval and witnesses.**

We need a comprehensive solution to fixing the changes made now that witnesses approve KELs before the delegator.
One potential option from Phil is that there is an additional round with the witnesses to forward the delegators KEL to the delegated (delegatee's) witnesses. It brings forth an additional discussion around is this functionality a witness should be doing (propagation). 

Associated issues:
- https://github.com/WebOfTrust/keripy/issues/762
- https://github.com/WebOfTrust/keripy/pull/766
- https://github.com/WebOfTrust/keripy/pull/680
- https://github.com/WebOfTrust/keripy/pull/842

**Comment by m00sey** [comment]
keripy:

**Open**
- https://github.com/WebOfTrust/keripy/pull/827

**Closed**
- https://github.com/WebOfTrust/keripy/pull/832
- https://github.com/WebOfTrust/keripy/pull/836
- https://github.com/WebOfTrust/keripy/pull/838
- https://github.com/WebOfTrust/keripy/pull/839
- https://github.com/WebOfTrust/keripy/pull/840
- https://github.com/WebOfTrust/keripy/pull/841
- https://github.com/WebOfTrust/keripy/pull/843

**Comment by henkvancann** [comment]
Kerific - @kordwarshuis
==> Made some changes that make the extension compliant with the Google Chrome Webstore again
Closed:
Wrong homepage URL and support URL in Chrome Store listing
#2 https://github.com/WebOfTrust/kerific/issues/2
Unwanted styling via Bootstrap
#1  https://github.com/WebOfTrust/kerific/issues/1

kerisse - @henkvancann
==> Aligned the design and navigation with KERIDoc and KERIGlos

keridoc - @henkvancann
==> Aligned the design and navigation with KERISSE and KERIGlos
Closed:
Reorganize the How We Did section into new directories
#6 https://github.com/WebOfTrust/keridoc/issues/6

**Comment by kentbull** [comment]
**Signify TS**:
- No new PRs
- High priority existing PRs to review:
  - https://github.com/WebOfTrust/signify-ts/pull/272
  - https://github.com/WebOfTrust/signify-ts/pull/271
  - https://github.com/WebOfTrust/signify-ts/pull/270 (was waiting on #267 which is now merged)
- no new issues

**CESRide**:
- No new PRs or issues

**Reply by daviddm** [reply]
https://github.com/WebOfTrust/signify-ts/pull/258 Can be merged

**Comment by rodolfomiranda** [comment]
**KERIA:**
PRs
- https://github.com/WebOfTrust/keria/pull/278
- https://github.com/WebOfTrust/keria/pull/275
- https://github.com/WebOfTrust/keria/pull/272
- https://github.com/WebOfTrust/keria/pull/271
- https://github.com/WebOfTrust/keria/pull/269

Draft PRs
- https://github.com/WebOfTrust/keria/pull/282
- https://github.com/WebOfTrust/keria/pull/281
- https://github.com/WebOfTrust/keria/pull/280
- https://github.com/WebOfTrust/keria/pull/267
